backend/app/Http/Controllers/AnswersController.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\ViewModels\Response;
+use App\Models\UserAnswerModel;
+use App\Models\AnswerModel;
+use App\Models\ChoiceModel;
+use App\Models\StatusModel;
+use App\Models\CategoryUsedModel;
+use Auth;
+use DB;
+
+class AnswersController extends Controller
+{
+    private $response;
+    public function __construct() {
+        $this->response = new Response();
+    }
+    
+    public function create_answer(Request $request) {
+        try {
+            DB::beginTransaction();
+
+            $answers = $request->progress;
+            $status_id = 0;
+
+            foreach ($answers as $answer) {
+                $answer = (object) $answer;
+                $choice = ChoiceModel::where([
+                    'question_id' => $answer->question_id,
+                    'choice_id' => $answer->choice_id
+                ])->first();
+
+                if ($choice) {
+                    $check_answer = AnswerModel::where([
+                        'choice_id' => $choice->choice_id
+                    ])->first();
+                    if ($check_answer) {
+                        $status = StatusModel::where('code', 'CRR')->first();
+                        $status_id = $status->status_id;
+                    } else {
+                        $status = StatusModel::where('code', 'IRR')->first();
+                        $status_id = $status->status_id;
+                    }
+                }
+
+                $create_answer = new UserAnswerModel();
+                $create_answer->category_id = $answer->category_id;
+                $create_answer->question_id = $answer->question_id;
+                $create_answer->choice_id = $answer->choice_id;
+                $create_answer->user_id = Auth::user()->user_id;
+                $create_answer->status_id = $status_id;
+                $create_answer->save();
```

Comment:
Instead of sending multiple queries at once.
How about just making one request to save to UserAnwerModal

https://laravel.com/docs/9.x/eloquent-relationships#the-create-method
Refer to `createMany` method